js/multiFileUpload.js

```diff
@@ -1,351 +1,390 @@
+                1: Qubit.multiFileUpload.i18nFilesSelected
+              },
+              // Upload status strings.
+              uploading: Qubit.multiFileUpload.i18nUploading,
+              complete: Qubit.multiFileUpload.i18nComplete,
+              uploadFailed: Qubit.multiFileUpload.i18nUploadFailed,
+              // Remove file hover text.
+              removeFile: Qubit.multiFileUpload.i18nRemoveFile,
+              // Main 'drop here' message.
+              dropPaste: Qubit.multiFileUpload.i18nDropFile,
+              filesUploadedOfTotal: {
+                0: Qubit.multiFileUpload.i18nFileUploadedOfTotal,
+                1: Qubit.multiFileUpload.i18nFilesUploadedOfTotal
+              },
+              dataUploadedOfTotal: Qubit.multiFileUpload.i18nDataUploadedOfTotal,
+              // When `showProgressDetails` is set, shows an estimation of how long the upload will take to complete.
+              xTimeLeft: Qubit.multiFileUpload.i18nTimeLeft,
+              uploadingXFiles: {
+                0: Qubit.multiFileUpload.i18nUploadingFile,
+                1: Qubit.multiFileUpload.i18nUploadingFiles
+              },
+              // Label cancel button.
+              cancel: Qubit.multiFileUpload.i18nCancel,
+              // Edit file hover text.
+              edit: Qubit.multiFileUpload.i18nEdit,
+              // Save changes button.
+              saveChanges: Qubit.multiFileUpload.i18nSave,
+              // Leave 'Add more' dialog.
+              back: Qubit.multiFileUpload.i18nBack,
+              // Edit Title dialog message.
+              editing: Qubit.multiFileUpload.i18nEditing,
+              failedToUpload: Qubit.multiFileUpload.i18nFailedToUpload,
             }
-          };
+          },
+          thumbnailWidth: Qubit.multiFileUpload.thumbWidth,
+          trigger: '#pick-files',
+          // Enable editing of field with id 'title' label: 'Title'
+          metaFields: [
+            { id: 'title', name: Qubit.multiFileUpload.i18nInfoObjectTitle },
+          ],
+        })
+        .use(Uppy.XHRUpload, {
+          endpoint: Qubit.multiFileUpload.uploadResponsePath,
+          formData: true,
+          method: 'post',
+          limit: 10,
+          fieldName: 'Filedata',
+          objectId: Qubit.multiFileUpload.objectId,
+        })
+        .on('upload-success', $.proxy(this.onUploadSuccess, this))
+        .on('complete', $.proxy(this.onComplete, this))
+        .on('file-added', $.proxy(this.onFileAdded, this))
+        .on('cancel-all', $.proxy(this.onCancelAll, this));
+    },
+
+    listen: function ()
+    {
+      // Intercept AtoM's Submit button.
+      this.$submitButton.on('click', $.proxy(this.onSubmitButton, this));
+      this.$retryButton.on('click', $.proxy(this.onRetryButton, this));
+    },
+
+    // Retry is available if some/all DO's do not successfully upload.
+    onRetryButton: function ()
+    {
+      this.uppy.retryAll().then((result) => {
```

Comment:
I see some ES6 arrow functions over here, something that I think we have not used before, but I think it's okay to have them nowadays, considering the only the close to death IE doesn't support them.

Reply:
My thinking as well.